### DISCUSSION TITLE
Apply one config to all users

### DISCUSSION THREAD

**Discussion by squromiv** [FreshRSS/FreshRSS#4834] [Q&A] [answered]
Is it possible somehow to apply one configuration (Display, Reading, Archiving) to all users by administrator? Except copying manually config.php to every user folder.

**Comment by squromiv** [comment, marked as answer]
Problem solved. Made multiple hard links to config.php file.

**Comment by Alkarex** [comment]
You can define settings in `./data/config-user.custom.php` which will be applied when creating new users.
There is no built-in option to globally override settings of existing users.
